### PR TITLE
feat(task): Remove token's from task structures

### PR DESCRIPTION
**Pull request by lyondhill**
We had previously removed token's from the task api but left the token in place in several locations in the stack.
Now we can cleanly remove the extra tokens.

### CODE REVIEW

File: http/swagger.yml [hunk 1]
```diff
@@ -8840,9 +8840,6 @@ components:
         flux:
           description: The Flux script to run for this task.
           type: string
-        token:
-          description: The token to use for authenticating this task when it executes queries.
-          type: string
       required: [flux, token]
```
Comment (kelwang) [inline http/swagger.yml]: does it need to be removed as well?

Comment (lyondhill) [reply http/swagger.yml]: I asked @jademcgough if we should remove it or deprecate we decided to just remove it.

Comment (kelwang) [reply http/swagger.yml]: I mean i think u also want to remove the token from the required, right

Comment (lyondhill) [reply http/swagger.yml]: oo your right Thanks!

**Pull request merged**